sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -340,6 +369,12 @@ case class FindInSet(left: Expression, right: Expression) extends BinaryExpressi
   }
 
   override def dataType: DataType = IntegerType
+
+  override def prettyName: String = "find_in_set"
+
+  override def sql: Option[String] = for {
```

Comment:
Then maybe simply returns a `String` and throw an exception if the expression is either not supported yet or inherently doesn't have a SQL representation (e.g. ScalaUDF)?